README.md

```diff
@@ -313,7 +349,52 @@ class Order extends Model
     ...
 }
 ```
+Create `OrderObserver.php` to track every order change.
+```php
+<?php
 
+namespace App\Observers;
+
+use App\Models\Order;
+use Dcodegroup\ActivityLog\Models\ActivityLog;
+
+class OrderObserver
+{
+    public function created(Order $model): void
+    {
+    $model->createActivityLog([
```

Comment:
In the code you have

```php
   $diff = data_get($description, 'diff', []);
```

But no example has this.

Are there only three parameters to pass in `Title` `Description` and `Diff`?

Reply:
I don't think we need the 'diff' column anymore. We previously made it available to support search functionality in search terms. However it was never implemented @lionslair , @andrewsirianni